src/main/java/com/google/devtools/build/lib/runtime/BuildSummaryStatsModule.java

```diff
@@ -161,6 +168,10 @@ public void buildComplete(BuildCompleteEvent event) {
       } else {
         reporter.handle(Event.info(Joiner.on(", ").join(items)));
         reporter.handle(Event.info(spawnSummary));
+        if (remoteMetrics != null) {
+          reporter.handle(Event.info(String.format("Network: %s remote upload, %s remote download",
```

Comment:
This doesn't take into account BES traffic. You should clarify that this only covers the remote execution / cache strategy.